src/Process/Factory/InitialTestsRunProcessFactory.php

```diff
@@ -59,7 +59,12 @@ public function createProcess(
         string $testFrameworkExtraOptions,
         array $phpExtraOptions,
         bool $skipCoverage,
+        bool $skipProgressBar,
     ): Process {
+        if ($skipProgressBar) {
+            $testFrameworkExtraOptions .= ' --no-progress';
+        }
```

Comment:
at best each test-adapter would set the necessary options on his own - no all test-runners have a option named `--no-progress`